### DISCUSSION TITLE
Any way to replace windows start menu with open-shell on a touch screen?

### DISCUSSION THREAD

**Discussion by phpjunkie420** [Open-Shell/Open-Shell-Menu#1880] [General]
I have a touch screen and when I swipe up from the bottom of the screen it shows the Windows Start Menu. Any way to force this to show Open-Shell instead?

https://github.com/Open-Shell/Open-Shell-Menu/assets/60593808/2870dda0-c43d-41e9-9274-a21b01fe3b20

**Comment by MmmZzzUhhm** [comment]
I also use a "Microsft Surface" and have been having the same problem for years. Perhaps the app needs to be improved, but I would appreciate your consideration. Thank you.
